src/main/java/leduc/command/EventCommand.java

```diff
@@ -69,7 +72,23 @@ else if(dateString[0].isBlank() || dateString[1].isBlank()){
             Date date1 = new Date(dateString[0]);
             Date date2 = new Date(dateString[1]);
             tasks.verifyConflictDate(date1, date2);
-            EventsTask newTask = new EventsTask(description, date1 , date2);
+            EventsTask newTask = null;
+            if (prioritySplit.length == 1){
+                newTask = new EventsTask(description, date1,date2);
+            }
+            else {
+                int priority = -1 ;
+                try{
+                    priority = Integer.parseInt(prioritySplit[1].trim());
+                }
+                catch(Exception e){
```

Comment:
same here, why not use PrioritizeFormatException

Reply:
A specified that PrioritizeLimitException represent a exception when the new priority is not an int or is greater than 9 or less than 0.

And PrioritizeFormatException is more for when the format for the PrioritizeCommand is not respected.

Reply:
ok